lib/components/CTACard/CTACard.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  Grid,
+  Paper,
+  Typography,
+  useTheme,
+} from '@apisuite/fe-base'
+
+import Link from '../../components/Link'
+import { CTACardProps } from './types'
+
+const CTACard: React.FC<CTACardProps> = ({
+  ctaCardButtonLabel,
+  ctaCardButtonLink,
```

Comment:
Rename props no need to call the prop `ctaCard[something]` if the component is already named `CTACard`.